content/projects/gsoc/2019/application.md

```diff
@@ -0,0 +1,144 @@
+---
+:layout: simplepage
+:title: "Jenkins GSoC application draft, 2019"
+:tags:
+- gsoc
+- gsoc2019
+---
+
+<!-- This file uses Markdown intentionally.
```

Comment:
@jenkins-infra/copy-editors and esp. @bitwiseman Would you be OK with using Markdown formatting for this page as an exception?